hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieDemo.java

```diff
@@ -115,6 +115,35 @@ public void testParquetDemo() throws Exception {
     testIncrementalHiveQueryAfterCompaction();
   }
 
+  @Test
+  public void testHFileDemo() throws Exception {
```

Comment:
this is good. but will add to our integration test runtime by a lot. we should disable this and enabled later. need a JIRA

Reply:
Ok.

Reply:
Filed https://issues.apache.org/jira/browse/HUDI-1229